Index: src/api/todolist-api.ts
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>import axios from \"axios\";\r\n\r\nconst instanse = axios.create({\r\n    baseURL: ' https://social-network.samuraijs.com/api/1.1/',\r\n    withCredentials: true\r\n})\r\nexport const TodolistApi = {\r\n    getTodolists() {\r\n        return instanse.get<TodolistType[]>('todo-lists')\r\n    },\r\n    createTodolist(title: string) {\r\n        return instanse.post<ResponseType<{item: TodolistType}>>('todo-lists', {title})\r\n    },\r\n    deleteTodolist(todoid: string) {\r\n        return instanse.delete<ResponseType>(`todo-lists/${todoid}`)\r\n    },\r\n    updateTodolistTitle(todolistId: string, title: string) {\r\n        return instanse.put<ResponseType>(`todo-lists/${todolistId}`, {title})\r\n    }\r\n}\r\n\r\ntype TodolistType = {\r\n    id: string,\r\n    title: string,\r\n    addedDate: Date,\r\n    order: number\r\n}\r\ntype ResponseType<T={}>={\r\n    resultCode: number\r\n    fieldsErrors: []\r\n    messages: string[],\r\n    data: T\r\n}\r\n\r\n\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/api/todolist-api.ts b/src/api/todolist-api.ts
--- a/src/api/todolist-api.ts	(revision b859ec2772537902c33c10c9ec57a2b160b51222)
+++ b/src/api/todolist-api.ts	(date 1693423765979)
@@ -4,6 +4,7 @@
     baseURL: ' https://social-network.samuraijs.com/api/1.1/',
     withCredentials: true
 })
+
 export const TodolistApi = {
     getTodolists() {
         return instanse.get<TodolistType[]>('todo-lists')
